src/Graphics/src/Graphics/Platforms/iOS/UIImageExtensions.cs

```diff
@@ -25,6 +26,24 @@ public static UIImage ScaleImage(this UIImage target, float maxWidth, float maxH
 			return ScaleImage(target, new CGSize(targetWidth, targetHeight), disposeOriginal);
 		}
 
+		internal static UIImage ResizeImageSource(this UIImage sourceImage, nfloat maxWidth, nfloat maxHeight, CGSize originalImageSize, bool shouldScaleUp = false)
+		{
+			if (sourceImage is null || sourceImage.CGImage is null)
```

Comment:
Can use the null-conditional operator to simplify the check:
`if (sourceImage?.CGImage is null)`